### ISSUE TITLE
Event when the carousel is fully loaded

### ISSUE THREAD

**Issue by dextherry** [OwlCarousel2/OwlCarousel2#2103]
I dont mean the event 'initialized' when the plugin is loaded, I mean, is there a event when the carousel is fully rendered, with all their elements and their css default values loaded, etc, ?

Regards

**Comment by samu1241** [comment]
Yes, I need this too. Can't find the right event when the navigation is rendered.
